meson_options.txt

```diff
@@ -73,3 +73,8 @@ option('allow_realloc',
   value : false,
   description : 'allow initial malloc size to grow to max line length (when using the heap)'
 )
+option('with_test_suite',
```

Comment:
Bit long, usually people just call this option `tests`

Reply:
:+1: @sage-etcher please make this change, and I'll merge this in.